thirdeye/thirdeye-frontend/app/pods/components/rootcause-chart-toolbar/component.js

```diff
@@ -71,6 +58,42 @@ export default Component.extend({
 
   granularity: null, // ""
 
+  //
+  // external (optional)
+  //
```

Comment:
Suggest to remove extra comment sign if it's only one line, otherwise, you can use multi-line comments /** */